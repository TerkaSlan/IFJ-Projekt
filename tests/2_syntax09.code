/*Should be error becasue of void variable*/

class Main {

	static void run() {
		void i;
	}
}