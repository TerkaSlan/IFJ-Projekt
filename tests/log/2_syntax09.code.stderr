ERROR: Syntactic - Line: 6 - Statment expected.
